From ca41c8ea8a9bffe151b5f0dba3541b77935ef000 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 12 Jul 2018 15:55:23 +0800
Subject: [PATCH 270/706] drm: hdp: Enable DPTX scrambling

This patch enables DPTX scramlbing. DPTX scrambles data symbols before
transmission.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 5c16316c142af50a16bd0e1a72a7276fa7d076a1)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-dp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-dp.c b/drivers/gpu/drm/imx/hdp/imx-dp.c
index 8d652f05d9b8..8f36a2c87e86 100644
--- a/drivers/gpu/drm/imx/hdp/imx-dp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-dp.c
@@ -90,7 +90,7 @@ void dp_mode_set(state_struct *state, int vic, int format, int color_depth, int
 	/* Number of lanes and SSC */
 	u8 num_lanes = 4;
 	u8 ssc = 0;
-	u8 scrambler = 0;
+	u8 scrambler = 1;
 	/* Max voltage swing */
 	u8 max_vswing = 3;
 	u8 force_max_vswing = 0;
-- 
2.17.1

